### PR TITLE
Julian changes

### PR DESCRIPTION
**Pull request by julian1**

### CODE REVIEW

**Comment by julian1** [comment]
@jonescc Don't merge yet.  comments welcome!

File: iso19139.mcp-2.0/schema/extensions/mcpExtensions.xsd [hunk 1]
```diff
@@ -229,6 +229,23 @@ Version 2.0 changes: mcp:taxonomicCoverage and mcp:dataParameter mods
 		<xs:attribute ref="gco:nilReason"/>
 	</xs:complexType>
 
+	<xs:element name="relatedTermURL" type="mcp:relatedTermURL_Type"/>
+
+	<xs:complexType name="relatedTermURL_Type">
```
Comment (jonescc) [inline iso19139.mcp-2.0/schema/extensions/mcpExtensions.xsd]: No need to define a complex type for this 

refer 

     <xs:element name="presentationURL" type="gmd:URL_PropertyType" minOccurs="0" maxOccurs="1"/>

above

defines presentationURL  to be of type gmd:URL_PropertyType

File: iso19139.mcp-2.0/schema/extensions/mcpExtensions.xsd [hunk 2]
```diff
@@ -229,6 +229,23 @@ Version 2.0 changes: mcp:taxonomicCoverage and mcp:dataParameter mods
...
+		<xs:sequence minOccurs="0">
+			<xs:element ref="gmd:URL"/>
+		</xs:sequence>
+	</xs:complexType>
+
+	<xs:element name="relatedTerm" type="mcp:relatedTerm_Type"/>
+
+	<xs:complexType name="relatedTerm_Type">
```
Comment (jonescc) [inline iso19139.mcp-2.0/schema/extensions/mcpExtensions.xsd]: sme here - can just use gco:CharacterString_PropertyType below

**Comment by jonescc** [comment]
@julian1, some issues with encoding throughout 

Types in the xsd should map to the types referred to in the uml diagram  - we should add additional types

**Comment by julian1** [comment]
@jonescc, I have removed the unwanted types.

**Comment by jonescc** [comment]
Looks good to me.

**Pull request merged**